db/migrate/20230518001155_add_assignee_column_to_chores.rb

```diff
@@ -0,0 +1,6 @@
+class AddAssigneeColumnToChores < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :chores, :assignee, null: true, foreign_key: {to_table: :users, on_delete: :cascade}
```

Comment:
When a user is deleted, I think making the assignee `NULL` would be better in my opinion. What do you think?